.pre-commit-config.yaml

```diff
@@ -75,3 +75,31 @@ repos:
           - flake8-print
         args:
           - --per-file-ignores=tests/*.py:T201 # prints are allowed in test files
+    # Use mypy for static type checking.
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: 'v0.991'
+    hooks:
+      - id: mypy
+        files: >
+          (?x)^(
+            src/snowflake/snowpark/modin/pandas/frontend/indexing.py|
```

Comment:
those paths are not correct any more after adding modin plugin. Can we fix them?